types/react-native-auth0/react-native-auth0-tests.tsx

```diff
@@ -285,27 +285,20 @@ auth0.credentialsManager.hasValidCredentials();
 auth0.credentialsManager.hasValidCredentials(123);
 
 function Test() {
-    const {
-        user,
-        error,
-        authorize,
-        clearSession,
-        getCredentials,
-        clearCredentials,
-        requireLocalAuthentication,
-    }: {
-        user: any;
-        error: BaseError | null;
-        authorize: (parameters: AuthorizeParams, options?: AuthorizeOptions) => Promise<void>;
-        clearSession: (parameters?: ClearSessionParams) => Promise<void>;
-        getCredentials: (scope?: string, minTtl?: number, parameters?: any) => Promise<Credentials>;
-        clearCredentials: () => Promise<void>;
-        requireLocalAuthentication: () => Promise<void>;
-    } = useAuth0();
+    const { isLoading, error, authorize, clearSession, getCredentials, clearCredentials, requireLocalAuthentication } =
+        useAuth0();
```

Comment:
I’d leave these lines as they were and update them to test `isLoading`, since they’re meant to test the return types from `useAuth0()`:

```suggestion
    const {
         user,
         error,
         isLoading,
         authorize,
         clearSession,
         getCredentials,
         clearCredentials,
         requireLocalAuthentication,
     }: {
         user: any;
         error: BaseError | null;
         isLoading: boolean;
         authorize: (parameters: AuthorizeParams, options?: AuthorizeOptions) => Promise<void>;
         clearSession: (parameters?: ClearSessionParams) => Promise<void>;
         getCredentials: (scope?: string, minTtl?: number, parameters?: any) => Promise<Credentials>;
         clearCredentials: () => Promise<void>;
         requireLocalAuthentication: () => Promise<void>;
     } = useAuth0();
```

Reply:
I don't think it's a very useful test, since it's just the exact same types specified in the declaration file. Anyone updating one will simply update the other, and then the tests will pass again (therefore not avoiding any regressions or issues).

Reply:
Fair enough